### ISSUE TITLE
0.1.1 release package does not include Go support?

### ISSUE THREAD

**Issue by nelhage** [bazelbuild/bazel#578]
Even though the 0.1.1 tag contains a [`tools/build_rules/go`](https://github.com/bazelbuild/bazel/tree/0.1.1/tools/build_rules/go) directory, the packaged release does not seem to include it:

```
$ unzip -l bazel-0.1.1-installer-darwin-x86_64.sh bazel-0.1.1-installer-linux-x86_64.sh | grep /go
$
```

Is this intentional? The [docs](http://bazel.io/docs/be/go.html) describe support as experimental, but make no note of requiring a source installation or other special consideration to make use of Go support.

**Comment by davidzchen** [comment]
This seems like a bug.

On a slightly related note, the docs on bazel.io are synced with `master` and are often ahead of the latest release. I think we should also add versioning for the Bazel docs for releases. This is also something that @damienmg brought up in a separate thread offline. I have opened #579 for that.

**Comment by damienmg** [comment]
This is because the BUILD file of the go rules aren't included in the filegroup for building bazel package. I let hanwen fix it :)

**Comment by damienmg** [comment]
Closing, resolved by 9f68000c9c2a93d24150066f70140b52de15a1cf and should be in next release.
